homeassistant/components/mealie/__init__.py

```diff
@@ -41,11 +49,20 @@ async def async_setup_entry(hass: HomeAssistant, entry: MealieConfigEntry) -> bo
     )
     try:
         about = await client.get_about()
+        version = create_version(about.version)
     except MealieAuthenticationError as error:
         raise ConfigEntryError("Authentication failed") from error
     except MealieConnectionError as error:
         raise ConfigEntryNotReady(error) from error
 
+    if not version.valid or version < MIN_REQUIRED_MEALIE_V:
+        LOGGER.error(
```

Comment:
We should not log in this case, but raise a `ConfigEntryError` instead (which supports translations, so we can drop the `OUTDATED_LOG_MESSAGE` constant).

Reply:
Also could we add a test for this case?

Reply:
Done, also added tests for auth & connection errors